scvi/external/stereoscope/_module.py

```diff
@@ -0,0 +1,240 @@
+from scvi.compose import AbstractVAE, auto_move_data, SCVILoss
```

Comment:
the imports should be sorted. https://pypi.org/project/isort/

just be sure to isort only the stereoscope directory